spec/services/claims/fetch_eligible_misc_fee_types_spec.rb

```diff
@@ -25,10 +25,14 @@
   context 'scheme 12 claim' do
     let(:claim) { create(claim_factory, :agfs_scheme_12) }
 
-    it 'returns misc fee types for AGFS scheme 10+ plus 12 without supplementary-only fee types' do
+    it 'returns misc fee types for AGFS scheme 12 without supplementary-only fee types' do
       is_expected.to have_at_least(1).items
       is_expected.to match_array Fee::MiscFeeType.agfs_scheme_12s.without_supplementary_only.map(&:unique_code)
     end
+
+    context 'when claim is a fixed fee type' do
```

Comment:
something missing here?

Reply:
yep, left in branch inadvertently. remove now